radis/lbl/loader.py

```diff
@@ -1091,6 +1103,7 @@ def fetch_databank(
             and dropping them with ``drop_columns``.
             If ``diluent`` then all additional columns required for calculating spectrum
             in that diluent is loaded.
+            Only ``'all'`` is available for kurucz linelists.
             Default ``'equilibrium'``.
```

Comment:
Rephrased with `parameter is ignored for Kurucz linelists`, as any other value just results in a warning and getting switched back to 'all', rather than raising an error